dali/dfu/dfurun.cpp

```diff
@@ -1226,13 +1227,16 @@ class CDFUengine: public CInterface, implements IDFUengine
                     if (!srcFile)
                         throw MakeStringException(-1,"Source file %s could not be found",tmp.str());
 
+                    srcName.set(tmp.str());
+                    srcFdesc.set(srcFile->getFileDescriptor());
```

Comment:
that will still leak.
getFilleDescriptor() returns a linked IFileDescriptor.
Owned::set also links.

you should use setown

Reply:
Changed